framework/src/modules/interoperability/mainchain/endpoint.ts

```diff
@@ -13,5 +13,51 @@
  */
 
 import { BaseEndpoint } from '../../base_endpoint';
+import { MainchainInteroperabilityStore } from './store';
+import { ImmutableStoreCallback, StoreCallback } from '../types';
+import { BaseInteroperableAPI } from '../base_interoperable_api';
+import { ModuleEndpointContext } from '../../../types';
 
-export class MainchainInteroperabilityEndpoint extends BaseEndpoint {}
+export class MainchainInteroperabilityEndpoint extends BaseEndpoint {
+	protected readonly interoperableCCAPIs = new Map<number, BaseInteroperableAPI>();
+
+	public constructor(moduleID: Buffer, interoperableCCAPIs: Map<number, BaseInteroperableAPI>) {
+		super(moduleID);
+		this.interoperableCCAPIs = interoperableCCAPIs;
+	}
+
+	public async getChainAccount(context: ModuleEndpointContext, chainID: Buffer) {
+		const interoperabilityStore = this.getInteroperabilityStore(context.getStore);
+		await interoperabilityStore.getChainAccount(chainID);
```

Comment:
Missing `return` here and on other places, I missed this in the last review for API but its fixed now